docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,132 @@
+# Use the Terraform module
+You can manage a Constellation cluster through Terraform.
+The necessary module package is available as part of the release and consists of a convenience module for each cloud service provider (`{csp}-constellation`) that combines the IAM (`infrastructure/{csp}/iam`), infrastructure (`infrastructure/{csp}`), and constellation (`constellation-cluster`) modules.
+
+## Prerequisites
+- a Linux / Mac operating system
+- a Terraform installation of at least `v1.4.4`
+
+## Quick setup
+The convenience module allows setting up a Constellation cluster with a single module. It's easiest to consume the module through a remote source, as shown below.
+This allows to upgrade the cluster to a newer Constellation version by simply updating the module source.
+
+In all examples, remember to replace `$VERSION` with the desired version, e.g., `v2.13.0`.
```

Comment:
I think this sentence needs now to be moved to the respective step and be rephrased accordingly